google/cloud/apigee/registry/applications/v1alpha1/registry_manifest.proto

```diff
@@ -0,0 +1,36 @@
+syntax = "proto3";
+
+package google.cloud.apigee.registry.applications.v1alpha1;
+
+option java_package = "com.google.cloud.apigee.registry.applications.v1alpha1";
+option java_multiple_files = true;
+option java_outer_classname = "RegistryManifestProto";
+option go_package = "github.com/apigee/registry/rpc;rpc";
+
+// Manifest represents the dependency relation between resources, which 
+// is used by the controller to keep the resources up-to-date.
+message Manifest {
+  // List of dependency relations.
+  repeated Resource resources = 1 [
+		(google.api.field_behavior) = REQUIRED
+	];
+}
+
+message Resource {
+	string pattern = 1 [
+		(google.api.field_behavior) = REQUIRED
+	];
+	string filter = 2;
+	repeated SourcePattern dependsOn = 3;
+	string updateAction = 4 [
+		(google.api.field_behavior) = REQUIRED
+	];
+}
+
+message SourcePattern {
+	string sourcePattern = 1 [
```

Comment:
Same here. Validation annotation.

Reply:
I think `pattern` would be better here, and it would be consistent with the definition of `pattern` in the `Resource` (or `GeneratedResource`) message.

Reply:
Yes, renamed to `pattern`